syft/sbom/sbom.go

```diff
@@ -31,21 +33,35 @@ type Descriptor struct {
 	Configuration interface{}
 }
 
-func AllCoordinates(sbom SBOM) []source.Coordinates {
```

Comment:
I updated this to a member function for consistency and to make this more discoverable. To me, it seems odd to define this in the same file to take the same data type